.github/workflows/pr-merge.yml

```diff
@@ -0,0 +1,77 @@
+name: PR Merge
+on:
+    pull_request:
+        types:
+          - closed
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+    check-pr-label:
+        if: ${{ github.event.pull_request.merged == true }}
+        runs-on: ubuntu-latest
+        
+        steps:
+          - uses: actions/checkout@v3
+          - name: Check for PR description label
+            id: check_pr_description_label
+            run: |
+              python scripts/process_message.py "${TEXT_BODY}" > processed_body.txt
+              processed_body=$(cat processed_body.txt)
+              proposed_version_impact=$(echo "$processed_body" | awk '/Major \\\(when you make incompatible API changes\\\)/,/Patch \\\(when you make backward compatible bug fixes\\\)/')
+              is_major=$(echo "$proposed_version_impact" | grep -qi '\[X\] Major'; echo $?)
```

Comment:
hmmm I am not fond of the fact that is_major is 0 when the impact is major and so on, since truthy values are usually 1.  
Could you flip the bit?

Reply:
This made the rest of the code rather unintuitive, like why is that the sum is 2 means it is correct?

Reply:
I used it as I was thinking of following the convention of the language, I will switch it over